includes/settings-cardknox.php

```diff
@@ -45,7 +45,7 @@
 		'token_key' => array(
 			'title'       => __( 'Cardknox Token Key', 'woocommerce-gateway-cardknox' ),
 			'type'        => 'text',
-			'description' => __( 'Get your iFields key from your Cardknox account.', 'woocommerce-gateway-cardknox' ),
+			'description' => __( 'Get your ifields key from your Cardknox account.', 'woocommerce-gateway-cardknox' ),
```

Comment:
This should not be changed. The requirement here was to say 'iFields' instead of 'API' which was done here #11.